pandas/core/window/numba_.py

```diff
@@ -0,0 +1,95 @@
+import types
+from typing import Callable, Dict, Optional, Tuple
+
+import numpy as np
+
+from pandas.compat._optional import import_optional_dependency
+
+
+def _generate_numba_apply_func(
```

Comment:
ok with this being non-private (e.g. generate_numba_apply_func), unless its *only* used in this module